contracts/HybridCustody.cdc

```diff
@@ -179,14 +184,14 @@ pub contract HybridCustody {
     TODO: Implement MetadataViews.Resolver and MetadataViews.ResolverCollection
     */
     pub resource Manager: ManagerPrivate, ManagerPublic {
-        pub let accounts: {Address: Capability<&{AccountPrivate, AccountPublic}>}
+        pub let accounts: {Address: Capability<&{AccountPrivate, AccountPublic, MetadataViews.Resolver}>}
         pub let ownedAccounts: {Address: Capability<&{Account, ChildAccountPublic, ChildAccountPrivate}>}
```

Comment:
Would we also want to include `MetadataViews.Resolver` for `ownedAccounts`?

Reply:
Yes! made a note of it, good catch